src/plugins/plot/configuration/PlotSeries.js

```diff
@@ -416,14 +421,14 @@ export default class PlotSeries extends Model {
    * when adding an array of points that are already properly sorted.
    *
    * @private
-   * @param {Object} point a telemetry datum.
-   * @param {Boolean} [appendOnly] default false, if true will append
+   * @param {Object} newData a telemetry datum.
+   * @param {Boolean} [sorted] default false, if true will append
    *                  a point to the end without dupe checking.
    */
-  add(point, appendOnly) {
+  add(newData, sorted = false) {
```

Comment:
very cool! i wonder how much this improves plot performance

Reply:
The logic already existed for arrays of historical data, I just renamed the variable for clarity, and applied the logic to arrays of batched real-time data as well. The renamed variable makes it clear that we are assuming the telemetry is sorted.